README.md

```diff
@@ -252,92 +248,21 @@ For example to add the [Gitea Git server](https://gitea.io/en-US/) to your Jenki
 
     jx create addon gitea
 
-This will: 
+This will:
 
 * install the Gitea Helm chart.
 * add Gitea as a Git server (via the `jx create git server gitea` command).
 * create a new user in Gitea (via the `jx create git user -n gitea` command).
 * create a new Git API token in Gitea (via the `jx create git token -n gitea -p password username` command).
-     
+
 ## Troubleshooting
 
 We have tried to collate common issues here with work arounds. If your issue isn't listed here please [let us know](https://github.com/jenkins-x/jx/issues/new).
- 
-### Cannot create cluster Minikube
```

Comment:
👍

Reply:
I think there is another issue just around this README update. https://github.com/jenkins-x/jx/issues/6799

Maybe while on it, we should add some other information into the README. @dennisnewel

Reply:
are you thinking about adding a deprecation/removal section or something? e.g. "We removed Minicube support in version 2.0.xxx"?

Reply:
@dennisnewel, I was more thinking of whether we should link to a more general FAQ. I would have thought we have a FAQ section in the online docs, but that does not seem to be the case. I don't think the README is a good location for this type of troubleshooting question. If there is no FAQ in the docs, we should create one there.

Regarding Minishift/Minikube, I don't think any deprecation section is needed. Definitely not in the README.